### PR TITLE
chore: copy over images for translated docs

### PR DESCRIPTION
**Pull request by nr-kkenney**
* added logic to copy over images for translated docs. also added in
paths for skipping over already copied images, & ignoring directories
with no images.

Resolves: #2288

### CODE REVIEW

File: scripts/actions/fetch-and-deserialize.js [hunk 1]
```diff
@@ -69,10 +107,9 @@ const fetchAndDeserialize = (accessToken) => async ({ locale, fileUris }) => {
   );
 
   createDirectories(files);
-
-  files.forEach((file) => writeSync(file, 'utf-8'));
+  writeFilesSync(files);
 };
 
 fetchAndDeserialize();
```
Comment (nr-kkenney) [inline scripts/actions/fetch-and-deserialize.js:113]: now that we have this imported into a test, this should be:
```javascript
if (process.env.CI) {
    fetchAndDeserialize();
}
```

**Review by nr-kkenney — COMMENTED** [review]
whoops 🙈

File: scripts/actions/fetch-and-deserialize.js [hunk 2]
```diff
@@ -69,10 +107,11 @@ const fetchAndDeserialize = (accessToken) => async ({ locale, fileUris }) => {
...
+  writeFilesSync(files);
 };
 
-fetchAndDeserialize();
+if (process.env.CI) {
+  fetchAndDeserialize();
+}
 
-module.exports = fetchAndDeserialize;
+module.exports = { writeFilesSync, fetchAndDeserialize };
```
Comment (zstix) [inline scripts/actions/fetch-and-deserialize.js:117]: If we change the export signature, I think we'll also need to change how it's imported in other files (`scripts/actions/check-job-prgress.js` comes to mind).

**Review by zstix — COMMENTED** [review]
Love the tests ✅ 
Love the comments ✅ 

Just had one minor thing about `module.exports`.

Hopefully we can avoid doing this work in the future (by referencing the english image rather than duplicating the files). That said, I think it's probably out of scope for this PR.

File: scripts/actions/fetch-and-deserialize.js (hunk 2 above)
Comment (nr-kkenney) [reply scripts/actions/fetch-and-deserialize.js:117]: oof. good catch.

**Review by aswanson-nr — APPROVED** [review]

**Pull request merged**